tests/unit-core/test-arraybuffer.c

```diff
@@ -151,17 +151,28 @@ static void test_write_with_offset (uint8_t offset) /**< offset for buffer write
   jerry_release_value (arraybuffer);
 } /* test_write_with_offset */
 
-static bool callback_called = false;
+static int callback_called = 0;
 static bool detach_free_callback_called = false;
```

Comment:
The detach_free_callback_called should be deleted and callback_called should be used instead